src/components/track/TrackPlayerContainer.jsx

```diff
@@ -18,12 +19,15 @@ const TrackPlayerContainer = ({
   track,
   isTwitter,
   backgroundColor,
+  did404
 }) => {
   const [didInitAudio, setDidInitAudio] = useState(false)
   const { popoverVisibility, setPopoverVisibility } = useContext(PauseContext)
 
   const onTrackEnd = useCallback(() => {
-    setPopoverVisibility(true)
+    if (flavor !== PlayerFlavor.TINY) {
+      setPopoverVisibility(true)
+    }
   }, [setPopoverVisibility])
```

Comment:
I'm surprised this doesn't complain about flavor not being captured, although I guess there's no reason it would ever change

Reply:
ah good call, we must not have the same lint stuff set up correctly here